gammapy/irf/effective_area.py

```diff
@@ -69,6 +69,25 @@ def plot(self, ax=None, energy=None, show_energy=None, **kwargs):
         return ax
 
     @classmethod
+    def from_parametrization(cls, energy, instrument='HESS'):
+        """Get parametrized effective area 
+
+        see :func:`~gammapy.irf.abramowski_effective_area`
```

Comment:
I removed the ``abramowksi_effective_area`` function. It was used on this docs page
http://docs.gammapy.org/en/latest/irf/effective_area.html
I moved the example show there to the docs string of the ``EffectiveAreaTable`` class and deleted this docs page. I think the same should be done for these pages
http://docs.gammapy.org/en/latest/irf/psf.html
http://docs.gammapy.org/en/latest/irf/energy_dispersion.html

@cdeil Objections?